itables/datatables_format.py

```diff
@@ -13,10 +13,11 @@ def _format_column(x):
     if dtype_kind in ["b", "i", "s"]:
         return x
 
-    try:
+    # If pandas version >= 0.25.0 use the new method format_array with parameter 'leading_space'
+    if pd.__version__ >= "0.25.0":
```

Comment:
I see that codecov does report your PR as covering that line at https://app.codecov.io/gh/mwouts/itables/pull/133

And it says at https://app.codecov.io/gh/mwouts/itables/blob/main/itables/datatables_format.py that the try/except approach does not cover the except paragraph

TBH I do not understand what is happening here - maybe I did not setup codecov correctly and it uses just one of the runs rather than the sum of all the runs?